lib/ansible/modules/cloud/ovirt/ovirt_instance_type.py

```diff
@@ -0,0 +1,592 @@
+            - "For I(host_passthrough) CPU type you need to set C(placement_policy) to I(pinned)."
+            - "If no value is passed, default value is set by oVirt/RHV engine."
+    rng_device:
+        description:
+            - "Random number generator (RNG). You can choose of one the following devices I(urandom), I(random) or I(hwrng)."
+            - "In order to select I(hwrng), you must have it enabled on cluster first."
+            - "/dev/urandom is used for cluster version >= 4.1, and /dev/random for cluster version <= 4.0"
+    rng_bytes:
+        description:
+            - "Number of bytes allowed to consume per period."
+    rng_period:
+        description:
+            - "Duration of one period in milliseconds."
+    placement_policy:
+        description:
+            - "The configuration of the instance type's placement policy."
+            - "Placement policy can be one of the following values:"
+            - "C(migratable) - Allow manual and automatic migration."
+            - "C(pinned) - Do not allow migration."
+            - "C(user_migratable) - Allow manual migration only."
+            - "If no value is passed, default value is set by oVirt/RHV engine."
+    cpu_pinning:
+        description:
+            - "CPU Pinning topology to map instance type CPU to host CPU."
+            - "CPU Pinning topology is a list of dictionary which can have following values:"
+            - "C(cpu) - Number of the host CPU."
+            - "C(vcpu) - Number of the instance type CPU."
+    soundcard_enabled:
+        description:
+            - "If I(true), the sound card is added to the instance type."
+        type: bool
+    smartcard_enabled:
+        description:
+            - "If I(true), use smart card authentication."
+        type: bool
+    virtio_scsi:
+        description:
+            - "If I(true), virtio scsi will be enabled."
+        type: bool
+    io_threads:
+        description:
+            - "Number of IO threads used by instance type. I(0) means IO threading disabled."
+    ballooning_enabled:
+        description:
+            - "If I(true), use memory ballooning."
+            - "Memory balloon is a guest device, which may be used to re-distribute / reclaim the host memory
+               based on instance type needs in a dynamic way. In this way it's possible to create memory over commitment states."
+        type: bool
+extends_documentation_fragment: ovirt
+'''
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Create instance type
+- name: Create instance type
+  ovirt_instance_type:
+    state: present
+    name: myit
+    rng_device: hwrng
+    rng_bytes: 200
+    rng_period: 200
+    soundcard_enabled: true
+    virtio_scsi: true
+    boot_devices:
+      - network
+
+# Remove instance type
```

Comment:
Also in the same follow up PR could you please use `- name:` rather than comments. Thanks in advance.